x/operator/keeper/operator_slash_state.go

```diff
@@ -0,0 +1,115 @@
+package keeper
```

Comment:
Please add comments for these functions, which should include when they are called by which module.

Reply:
Okay